### PR TITLE
use carbon for nice human dates. add country name in full

### PR DESCRIPTION
**Pull request by dmorley**
needed more granular than months to make this work right. Not sure why country name happened at same time, my brain is goofy.

https://github.com/diasporg/Poduptime/issues/150

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -143,7 +144,7 @@
 <script src="bower_components/tablesorter/dist/js/jquery.tablesorter.min.js"></script>
 <script src="js/podup.js"></script>
 <script src="bower_components/tether/dist/js/tether.min.js"></script>
-<script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+<script src="bower_components/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
```
[the remark this answers is not in the document]
Comment (dmorley) [reply index.php]: Yea, I didn't want to keep changing it over and over, as we merge all these PRs things will align I hope.

**Comment by noplanman** [comment]
Reminder to add changelog entries for PRs :innocent:

**Comment by dmorley** [comment]
I did all branches off develop and it does not have one so I was not sure how to do it without a mess :/
Maybe my workflow should branch elsewhere?

**Comment by noplanman** [comment]
Just add an entry to the changelog in each PR, then as they are merged, you can simply rebase develop onto the other PRs (which you'll need to do anyway) and merge the changelog entries.

**Pull request merged**